php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S4830.html

```diff
@@ -1,28 +1,23 @@
-<p>Disabling SSL/TLS certificates chain of trust verification is similar to trust every one in the chain and so to expose the application to man-in-the-middle (MITM) attacks.</p>
+<p>Disabling SSL/TLS certificates chain of trust verification is similar to trust every one in the chain and so to expose the application to
+man-in-the-middle (MITM) attacks.</p>
 <h2>Noncompliant Code Example</h2>
 <pre>
 curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, FALSE);
 curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, TRUE); // Noncompliant; TRUE is casted to 1 which is not a secure configuration
-curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, '0');
+curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, 0);
 
 // and/or
 
 curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, FALSE);
-curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, '0');
+curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, 0);
 </pre>
 <h2>Compliant Solution</h2>
 <pre>
-curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, '2'); // Compliant; default value is 2 to "check the existence of a common name and also verify that it matches the hostname provided" according to PHP's documentation
+curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, 2); // Compliant; default value is 2 to "check the existence of a common name and also verify that it matches the hostname provided" according to PHP's documentation
 
 // and/or
 
 curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, TRUE); // Compliant; default value is TRUE
-curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, '1');
+curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, 1);
```

Comment:
the test cases may have to be updated?

Reply:
tests already cover both number and string